nexus/src/external_api/http_entrypoints.rs

```diff
@@ -7562,6 +7563,28 @@ async fn group_list_v1(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Fetch group
+#[endpoint {
+    method = GET,
+    path = "/v1/groups/{group}",
+    tags = ["silos"],
```

Comment:
I wonder about the tag here. Isn't anything to be changed in this PR, but should silo scoped resources be listed in the silos tag?

Reply:
I agree that it's weird. Perhaps even weirder: it seems to _only_ be silo-scoped resources that get this tag. All the `/system/silos/*` endpoints for working with silos themselves are tagged `system`. We could probably stand to rework all the tags.

```
API operations found with tag "silos"
OPERATION ID                             URL PATH
group_list                               /groups
group_list_v1                            /v1/groups
group_view                               /v1/groups/{group}
policy_update                            /policy
policy_update_v1                         /v1/policy
policy_view                              /policy
policy_view_v1                           /v1/policy
user_list                                /users
user_list_v1                             /v1/user
```